src/common/components/game/Hand.js

```diff
@@ -65,12 +66,12 @@ export default class Hand extends Component {
             card={card}
             idx={idx}
             margin={idx < numCards - 1 ? cardMargin : 0}
-            rotation={this.props.curved ? rotationDegs : 0}
+            rotation={(this.props.sandbox && this.props.opponent) ? 180 : (this.props.curved ? rotationDegs : 0)}
```

Comment:
This is a bit complex as a line of logic for being a one-liner. I think it needs its own function.

Reply:
I'll clean it up a bit.